src/types/index.d.ts

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+
+/**
+ * Little typescript contribution from KR Tirtho...
+ * Hope you will accept this ❤
+ * 
+ * What works?
+ * Fully supported with typescript (after builds)
+ * PropTypes intellisense in (React) <typescript> projects
+ * Warns if user provides wrong <types> in props 
+ * Short description of every props
+ * 
+ * Finally, thanks for making this library. It helped me a lot in one of my private project.....
+ */
+
+declare module "react-loader-spinner" 
+{
+    // Main Interface for the props
+    interface LoaderProps<T = {}> {
+    /**
+     * @property _visible_ | Show/ Hide the loader as required
+     * @default  false
+     */
+    visible?: boolean;
+    /**
+     * @property _type_ | Type of spinner you want to display
+     * @default Audio
+    */
+      type: 
+      "Audio"
+      |"BallTriangle"
+      |"Bars"
+      |"Circles"
+      |"Grid"
+      |"Hearts"
+      |"Oval"
+      |"Puff"
+      |"Rings"
+      |"TailSpin"
+      |"ThreeDots"
+      |"Watch"
+      |"RevolvingDot"
+      |"Triangle"
+      |"Plane"
+      |"MutatingDots"
+      |"CradleLoader";
+      /**
+       * @property _height_ | Height prop define the height of the svg spinner. 
+       * @default 80px
+        */
+      height?: number;
+      /**
+       * @property _width_ | Width prop define the width of the spinner.
+       * @default 80px
+        */
+      width?: number;
```

Comment:
[Both height and width can be strings](https://github.com/mhnpd/react-loader-spinner/issues/59).

Reply:
Okay, give me just two mins😄

Reply:
Yeah Fixed👍